app/assets/javascripts/backbone/views/project_step_modal_view.coffee

```diff
@@ -68,3 +72,10 @@ class MS.Views.ProjectStepModalView extends Backbone.View
   runAndResetDoneCallback: ->
     @done()
     @done = (->) # Reset to empty function.
+
+  showHideStartDate: ->
+    @precedingStepId = @$('#project_step_schedule_parent_id').val()
+    if @precedingStepId
+      @$('.form-group.project_step_scheduled_start_date').hide()
+    else
+      @$('.form-group.project_step_scheduled_start_date').show()
```

Comment:
Very nice. I think in future we should add a jquery method to all projects called `showhide` or something, that takes a boolean param so that we can avoid this repetition. But this works great for now.